dynawo/sources/Modeler/Common/DYNCompiler.cpp

```diff
@@ -74,8 +73,6 @@ using std::vector;
 using boost::dynamic_pointer_cast;
 using boost::lexical_cast;
 using boost::shared_ptr;
-using boost::unordered_map;
```

Comment:
Why removing the using instead of modifying them?

Reply:
I feel that the "using" keyword makes the code less clear

Reply:
This is creating a lot of differences that will make harder to explore the history of this file, I'm not fond of it

Reply:
I put back the using in that case

Reply:
I've just realized the using keyword can create confusion in the files which contains both boost containers and std containers. I'm not sure it's a good idea to restore these changes.